site/docs/tools/deltalake/spark.md

```diff
@@ -0,0 +1,191 @@
+# Spark via Delta Lake
+
+!!! note
+    Detailed steps on how to set up Pyspark + Delta Lake + Nessie with Python is available on [Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=colab%2Fnessie-delta-spark-setup.ipynb).
```

Comment:
The "Binder" link is broken now. (colab/ directory's gone)

Reply:
Can you add the DL demo link here?

Reply:
done